Test can't find inum64

I don't know why, but when i'm compiling the test objects, it isn't seeing the `inum64` headers. Meson setup output:
```zsh
$ env PKG_CONFIG_PATH=.:/Users/dannys4/installs/INUM64/1.0.3/lib/pkgconfig CC=clang CXX=clang++ meson setup --wipe build --prefix /Users/dannys4/installs/REALPAVER -DINTERVAL_LIB=INUM64 -DLP_LIB=Gurobi

The Meson build system
Version: 1.9.0
Source dir: /Users/dannys4/git-repos/realpaver
Build dir: /Users/dannys4/git-repos/realpaver/build
Build type: native build
Project name: Realpaver
Project version: 1.1.1
C++ compiler for the host machine: clang++ (clang 19.1.7 "clang version 19.1.7")
C++ linker for the host machine: clang++ ld64 955.13
Host machine cpu family: aarch64
Host machine cpu: aarch64
Found pkg-config: YES (/opt/homebrew/bin/pkg-config) 2.5.1
Run-time dependency gurobi found: YES 12.0.3
Run-time dependency inum64 found: YES 1.0.3
Library m found: YES
Has header "dlfcn.h" : YES 
Has header "inttypes.h" : YES 
Has header "stdint.h" : YES 
Has header "stdio.h" : YES 
Has header "stdlib.h" : YES 
Has header "strings.h" : YES 
Has header "string.h" : YES 
Has header "sys/stat.h" : YES 
Has header "sys/types.h" : YES 
Has header "unistd.h" : YES 
Program flex found: YES (/Users/dannys4/miniconda3/envs/realpaver/bin/flex)
Program bison found: YES (/Users/dannys4/miniconda3/envs/realpaver/bin/bison)
Configuring configure.hpp using configuration
Program doxygen found: NO
doc/meson.build:20: WARNING: Doxygen not found, no documentation generated!
Program pkg-config found: YES (/opt/homebrew/bin/pkg-config)
Build targets in project: 57

Realpaver 1.1.1

  User defined options
    INTERVAL_LIB: INUM64
    LP_LIB      : Gurobi
    prefix      : /Users/dannys4/installs/REALPAVER

Found ninja-1.13.1 at /Users/dannys4/miniconda3/envs/realpaver/bin/ninja

$ cd build
$ meson compile

(lots of warnings from DEBUG flags)

[131/215] Compiling C++ object test/ctc_var_3b_test.p/ctc_var_3b_test.cpp.o
FAILED: [code=1] test/ctc_var_3b_test.p/ctc_var_3b_test.cpp.o 
clang++ -Itest/ctc_var_3b_test.p -Itest -I../test -Isrc -I../src -fdiagnostics-color=always -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_FAST -Wall -Winvalid-pch -std=c++17 -O0 -g -ftree-vectorize -fPIC -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -fmessage-length=0 -isystem /Users/dannys4/miniconda3/envs/realpaver/include -D_FORTIFY_SOURCE=2 -isystem /Users/dannys4/miniconda3/envs/realpaver/include -MD -MQ test/ctc_var_3b_test.p/ctc_var_3b_test.cpp.o -MF test/ctc_var_3b_test.p/ctc_var_3b_test.cpp.o.d -o test/ctc_var_3b_test.p/ctc_var_3b_test.cpp.o -c ../test/ctc_var_3b_test.cpp
In file included from ../test/ctc_var_3b_test.cpp:1:
In file included from ../src/realpaver/ContractorConstraint.hpp:24:
In file included from ../src/realpaver/Constraint.hpp:24:
In file included from ../src/realpaver/Term.hpp:24:
In file included from ../src/realpaver/DomainBox.hpp:24:
In file included from ../src/realpaver/Domain.hpp:24:
In file included from ../src/realpaver/Interval.hpp:25:
src/realpaver/IntervalImpl.hpp:24:10: fatal error: 'inum64/inum64.hpp' file not found
   24 | #include "inum64/inum64.hpp"
      |          ^~~~~~~~~~~~~~~~~~~
1 error generated.

```

--- a/test/meson.build
+++ b/test/meson.build
@@ -81,6 +81,7 @@ ctc_var_3b_test = executable(
   'ctc_var_3b_test',
   sources: 'ctc_var_3b_test.cpp',
   link_with: realpaver_lib,
+  dependencies: deps_lib,
   include_directories: ['../src'],
 )
 test('Test of var3B contractor', ctc_var_3b_test)
@@ -208,6 +209,7 @@ piecewise_test = executable(
   'piecewise_test',
   sources: 'piecewise_test.cpp',
   link_with: realpaver_lib,
+    dependencies: deps_lib,
   include_directories: ['../src'],
 )
 test('Test of piecewise', piecewise_test)

